diff -u -r -U5 --no-dereference ninja-1.7.1/src/subprocess_test.cc ninja-1.7.1.resource-intensive-test/src/subprocess_test.cc
--- ninja-1.7.1/src/subprocess_test.cc	2016-04-28 10:56:44.000000000 -0400
+++ ninja-1.7.1.resource-intensive-test/src/subprocess_test.cc	2016-07-24 17:29:42.783820341 -0400
@@ -214,11 +214,11 @@
   }
 }
 
 // OS X's process limit is less than 1025 by default
 // (|sysctl kern.maxprocperuid| is 709 on 10.7 and 10.8 and less prior to that).
-#if !defined(__APPLE__) && !defined(_WIN32)
+#if 0 && !defined(__APPLE__) && !defined(_WIN32)
 TEST_F(SubprocessTest, SetWithLots) {
   // Arbitrary big number; needs to be over 1024 to confirm we're no longer
   // hostage to pselect.
   const unsigned kNumProcs = 1025;
 
